cfme/markers/isolation.py

```diff
@@ -0,0 +1,26 @@
+"""Marker for browser isolation on specific tests"""
+import pytest
+
+from cfme.test_framework.browser_isolation import browser_implementation_quits
+from cfme.utils.log import logger
+
+
+# Add Marker
+def pytest_configure(config):
+    config.addinivalue_line('markers', 'browser_isolation: Mark a test case for browser isolation')
+
+
+@pytest.mark.hookwrapper(tryfirst=True)
+def pytest_runtest_setup(item):
+    if item.get_marker('browser_isolation'):
+        logger.info('Browser isolation for marker in setup')
+        browser_implementation_quits(item)
+    yield
```

Comment:
**Suggestion:**
Blank `yield` is not require.

Reply:
Sure do in these hookwrappers.  Without yielding, you'll see this:

```
___________________________ ERROR at setup of test_login_evm_group[ldap-ad_dev_2012-cn-ldapuser1] ____________________________

item = <Function 'test_login_evm_group[ldap-ad_dev_2012-cn-ldapuser1]'>

    @pytest.mark.hookwrapper(tryfirst=True)
    def pytest_runtest_setup(item):
        """Pytest hook ensuring that failures caused by NotImplementedError show up as skips instead"""
        outcome = yield
        try:
>           outcome.get_result()
E           TypeError: 'NoneType' object is not an iterator

```

Reply:
Thanks @mshriver :smile: